pkg/cloud/aws/s3selectintegration.go

```diff
@@ -178,6 +185,17 @@ func (s3si *S3SelectIntegration) GetCloudCost(
 					labels[labelName] = value
 				}
 			}
+			for _, awsLabelColumnName := range awsLabelColumns {
+				// remove quotes
+				labelName := strings.TrimPrefix(awsLabelColumnName, `s."`)
+				labelName = strings.TrimSuffix(labelName, `"`)
+				// partially remove prefix leaving "aws:"
+				labelName = strings.TrimPrefix(labelName, "resourceTags/")
+				value := GetCSVRowValue(row, columnIndexes, awsLabelColumnName)
+				if value != "" {
+					labels[labelName] = value
+				}
```

Comment:
tests please!

Reply:
These changes relate to methods that integrate with s3 and Athena. Unit tests are not applicable in this case.